corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fake_formplayer.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Generates mock formplayer responses. Use as a fake for queryFormplayer.
+ *
+ * This does not support many of the options that can be passed to the real queryFormplayer.
+ * It primarily fakes the selections logic.
+ *
+ * This contains one app, which has the following structure:
+ *      m0: a menu that does not use cases
+ *          m0-f0
+ *      m1: a menu that uses cases
+ *          m1-f0: a form that updates a case
+ *  No menus use display-only forms.
+ */
+hqDefine("cloudcare/js/formplayer/spec/fake_formplayer", function () {
+    var AssertProperties = hqImport("hqwebapp/js/assert_properties"),
```

Comment:
general PR nit: prefer `let` over `var` since it creates variables with reduced scope

Reply:
did a find and replace in the `formplayer/spec` directory: 50d4139a724